app/app.gradle.kts

```diff
@@ -60,11 +55,11 @@ if (project.hasProperty("jenkinsBuild") || project.hasProperty("headless")) {
     tasks {
         "junitPlatformTest"(JavaExec::class) {
             jvmArgs = listOf(
-                "-Djava.awt.headless=true",
-                "-Dtestfx.robot=glass",
-                "-Dtestfx.headless=true",
-                "-Dprism.order=sw",
-                "-Dprism.text=t2k"
+                    "-Djava.awt.headless=true",
+                    "-Dtestfx.robot=glass",
+                    "-Dtestfx.headless=true",
+                    "-Dprism.order=sw",
+                    "-Dprism.text=t2k"
```

Comment:
No, the indentation rule isn't correctly implemented yet.
https://github.com/shyiko/ktlint/issues/76